lib/puppet/rest/routes.rb

```diff
@@ -35,11 +36,13 @@ def self.get_certificate(client, name)
     # HTTP client. Accepts a block to stream responses to disk.
     # @param [Puppet::Rest::Client] client the HTTP client to use to make the request
     # @param [String] name the crl to fetch
+    # @param [String] last_modified the httpdate of when the agent crl was last modified
```

Comment:
I think that this should be a `Time` object instead of a string; at least, that is what is returned by the `mtime` method for `File`.

Reply:
Yep!  Thank you!